### PR TITLE
[AVFoundation] Add missing ios api.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/avfoundation.cs [hunk 1]
```diff
@@ -3547,6 +3547,10 @@ interface AVUrlAsset {
 		[iOS (10,0), TV (10,0), Mac (10,12)]
 		[NullAllowed, Export ("assetCache")]
 		AVAssetCache Cache { get; }
+
+		[iOS (10, 0), TV (10, 0), NoWatch, NoMac]
+		[Field ("AVURLAssetAllowsCellularAccessKey")]
+		NSString AVURLAssetAllowsCellularAccessKey { get; }
```
Comment (dalexsoto) [inline src/avfoundation.cs]: Better name? maybe just `AllowsCellularAccessKey`

Comment (mandel-macaque) [inline src/avfoundation.cs]: True, I had that in my mind when doing the binding, I though I did change it.

**Comment by rolfbjarne** [comment]
LGTM 👍

**Comment by spouliot** [comment]
@mandel-macaque please include the bug# in the PR, that helps validation ;-)
https://bugzilla.xamarin.com/show_bug.cgi?id=43801

**Pull request merged**